contracts/Foundation.sol

```diff
@@ -24,7 +24,7 @@ contract Foundation is System {
     if (ret) {
       emit fundSuccess(payee, amount);
     } else {
-      emit fundFailed(payee, amount, address(this).balance);
+      emit fundFailed(payee, amount, address(this).balance); //@openissue
```

Comment:
this function does not revert on call() failure leaving GovHub's execute() no way of knowing of the failure. 
what's the logic here?

Reply:
just note that this problem is not related to using send() vs. call(), it's a DoS issue, not a reentry one.
anyway - we can leave it as-is since this is also a PM issue which we agreed not no handle now